packages/web-react/src/components/Grid/README.md

```diff
@@ -58,6 +58,39 @@ Advanced example usage:
 </Grid>
 ```
 
+### ⚠️ DEPRECATION NOTICE
+
+GridSpan component is deprecated and will be removed in the next major version. Use `GridItem` instead.
+
+[What are deprecations?][readme-deprecations]
+
+#### Migration Guide
```

Comment:
Should this be repeated in this readme as well? I mean the same text is in deprecations so maybe just linking it?

Reply:
As agreed in person, please keep the migration instructions only in the DEPRECATIONS file. We can leave the deprecation notice here.